ShaderTool/Command/Texture.cs

```diff
@@ -54,7 +54,7 @@ class Texture {
                     continue;
                 }
 
-                string fileName = Path.GetFileNameWithoutExtension(texturePath);
+                string fileName = Path.GetFileNameWithoutExtension(texturePath).Replace(" ", "_");
```

Comment:
should probably just check with AssertName() instead of automatically replacing spaces with underscores

Reply:
This should be done because it affects other inputs to and therefore spaces in file names are a bad idea (See settex for further info)